src/debug.h

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for license information.
+ */
+
+#ifndef __DEBUG_H__
+#define __DEBUG_H__
+
+#include <stdbool.h>
+#include <stdio.h>
+
+extern bool debug;
```

Comment:
NIT: Should this be named `DEBUG` since it's globally defined? Or does that only apply to global statics?

Reply:
ALLCAPS are typically macros in C.  Some code styles use g_ or global_ to help indicate as such.